### DISCUSSION TITLE
求助，如何使用paddleocr识别车牌

### DISCUSSION THREAD

**Discussion by xiaoyue-shi** [PaddlePaddle/PaddleOCR#14244] [Q&A]
求助，如何使用paddleocr识别车牌，我使用ocr = PaddleOCR(use_angle_cls=True,use_gpu=False,lang='ch') 识别效果很差

**Comment by GreatV** [comment]
你得收集数据，微调一个模型

**Reply by yinlin712** [reply]
识别车牌主要的工作在目标检测吧，ocr占的比例蛮小的
